R/lint-posts.R

```diff
@@ -198,14 +185,16 @@ rolint_figure_shortcode <- function(post_xml) {
     xml2::xml_find_all("//image") -> images
 
   if (length(images) == 0) {
-    return(NULL)
+    usethis::ui_done("Found no image not using the Hugo figure shortcode.")
```

Comment:
Otherwise returns both the 'correct' statement and the 'incorrect' statement

```suggestion
    usethis::ui_done("Found no image not using the Hugo figure shortcode.")
    return(NULL)
```